### ISSUE TITLE
SimpleRatingBar lacks accessibility support

### ISSUE THREAD

**Issue by hanghildegard** [FlyingPumba/SimpleRatingBar#30]
The rating bar is not recognised by accessibility helpers such as TalkBack and thus is not selectable. Users should be able to select the rating bar when using touch exploration and they should be able to get audible feedback about it as well as about the rating value selected.

As this rating bar is a custom view with virtual child views, the virtual view hierarchy (in the accessibility node provider, for example) and the accessibility events applicable to the view have to be individually defined.

I would really appreciate if you could add this as accessibility is quite an important issue.

**Comment by FlyingPumba** [comment]
Hi, sorry for the late response. I agree that this is an important issue, but I have never worked with the accessibility API. I will be looking to the [docs](https://developer.android.com/guide/topics/ui/accessibility/custom-views.html) and see if I can come up with a fix (PRs are welcomed, though).
